### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by Cloudage**
guess it should be a typo at Stack.h line 460.

### CODE REVIEW

**Comment by alexeevdv** [comment]
@vinniefalco what is the problem to fix this bug? looks like this bug exist more than an year for now :)

**Comment by vinniefalco** [comment]
@alexeevdv Are you asking me why I'm not fixing it? Because I am working on a commercial project and I am not set up to do LuaBridge development. I would need to add unit tests and revise the library significantly. I can't give the library the attention that it needs, so the safest course of action is to simply not touch it.

File: Source/LuaBridge/detail/LuaRef.h [hunk 1]
```diff
@@ -38,9 +38,9 @@
         ...
         t ["k"] = Nil(); // assign nil
 */
-struct Nil
-{
-};
+//struct Nil
```
Comment (dmitry-t) [inline Source/LuaBridge/detail/LuaRef.h]: Commenting out does not work. Please keep the code clean.

**Comment by Ryanel** [comment]
#130 merged. Lua docs state that the functions are functionally identical (lua_pushlstring is called by lua_pushstring, just with a strlen)

**Pull request closed without merging**